naiserator-dev.yaml

```diff
@@ -62,4 +62,5 @@ spec:
       value: jdbc:postgresql://b27dbvl013.preprod.local:5432/egenmeldt-sykmelding-backend
     - name: MOUNT_PATH_VAULT
       value: /postgresql/preprod-fss
-
+    - name: REGISTER_BASE_PATH
```

Comment:
Vi kommer antageligvis til å integrere med flere registre. Kanskje kalle denne AAREG_BASE_PATH i stedet? :) Samme gjelder prod og environment :)

Reply:
Ah, du bruker den for ereg og ja. La den bare stå sånn :)